borsh/src/de/mod.rs

```diff
@@ -25,11 +25,13 @@ use crate::maybestd::{rc::Rc, sync::Arc};
 
 mod hint;
 
-const ERROR_NOT_ALL_BYTES_READ: &str = "Not all bytes read";
-const ERROR_UNEXPECTED_LENGTH_OF_INPUT: &str = "Unexpected length of input";
-const ERROR_OVERFLOW_ON_MACHINE_WITH_32_BIT_ISIZE: &str = "Overflow on machine with 32 bit isize";
-const ERROR_OVERFLOW_ON_MACHINE_WITH_32_BIT_USIZE: &str = "Overflow on machine with 32 bit usize";
-const ERROR_INVALID_ZERO_VALUE: &str = "Expected a non-zero value";
+pub const ERROR_NOT_ALL_BYTES_READ: &str = "Not all bytes read";
+pub const ERROR_UNEXPECTED_LENGTH_OF_INPUT: &str = "Unexpected length of input";
+pub const ERROR_OVERFLOW_ON_MACHINE_WITH_32_BIT_ISIZE: &str =
+    "Overflow on machine with 32 bit isize";
+pub const ERROR_OVERFLOW_ON_MACHINE_WITH_32_BIT_USIZE: &str =
+    "Overflow on machine with 32 bit usize";
+pub const ERROR_INVALID_ZERO_VALUE: &str = "Expected a non-zero value";
```

Comment:
It might be not great to make these strings to be part of the public interface. What was the motivation to make them `pub`?